src/dotnet/APIView/APIViewWeb/Helpers/LanguageServiceHelpers.cs

```diff
@@ -42,6 +42,27 @@ public static string MapLanguageAlias(string language)
             return SupportedLanguages.Where(lang => lang.Equals(language, StringComparison.OrdinalIgnoreCase)).FirstOrDefault() ?? language;
         }
 
+        public static string GetLanguageAliasForCopilotService(string language)
```

Comment:
this mapping means that the review agent understands `clang` instead of `c` as language?